File: laminatoria-client/src/app/classes/orderItem.ts
import { Product } from './product'

export class OrderItem {
	id: number
	product: Product
	amount: number
	additionalPropValue: string
	orderId: number
	sumPrice: number

	constructor(
		id: number,
		product: Product,
		amount: number,
		additionalPropValue: string,
		orderId: number,
		sumPrice: number
	) {
		this.id = id
		this.product = product
		this.amount = amount
		this.additionalPropValue = additionalPropValue
		this.orderId = orderId
		this.sumPrice = sumPrice
	}
}

File: Laminatoria/Models/OrderItem.cs
﻿namespace Laminatoria.Models
{
    public class OrderItem
    {
        public int Id { get; set; }
        public int Amount { get; set; }
        public string AdditionalPropValue { get; set; }
        public int? ProductId { get; set; }
        public Product? Product { get; set; }
        public int OrderId { get; set; }
        public Order Order { get; set; }
        public decimal SumPrice { get; set; }
    }
}
